src/bindings/python/src/openvino/properties/intel_gpu/hint/__init__.py

```diff
@@ -3,10 +3,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Properties
-from openvino._pyopenvino.properties.intel_gpu.hint import queue_throttle
-from openvino._pyopenvino.properties.intel_gpu.hint import queue_priority
-from openvino._pyopenvino.properties.intel_gpu.hint import host_task_priority
-from openvino._pyopenvino.properties.intel_gpu.hint import available_device_mem
+import openvino._pyopenvino.properties.intel_gpu.hint as __hint
+from openvino.properties.properties import __make_properties
```

Comment:
```suggestion
from openvino.properties._properties import __make_properties
```
Would it be possible to make it more "private"? I know most of the functions inside are done this way but either changing it to `_properties` or other name (i.e. `impl`/`details`) would help to distinguish module and implementation from each other.

Reply:
marked it as `_properties`.